www/js/overlay.js

```diff
@@ -13,8 +13,12 @@ const { store } = remote.require('./app');
 const analytics = remote.getGlobal('analytics');
 
 const { overlayVars } = store.get('obs').overlayPrefs;
+let overlayCast = store.getUserPref('app.overlay-cast');
 
-const url = `http://${host}:${overlayPort}/`;
+const controlPanel = document.querySelector('.control-panel');
+const webview = document.createElement('webview');
+
+let url = `http://${host}:${overlayPort}/`;
```

Comment:
copy button's url doesn't get updated when initially off

Reply:
Yes, but it gets updated as soon as we click the on/off switch.